config-connector/solutions/sql/kpt/mysql-ha/Kptfile

```diff
@@ -0,0 +1,65 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: mysql-ha
+packageMetadata:
+  shortDescription: mysql high availability cluster
+openAPI:
+  definitions:
+    io.k8s.cli.setters.password:
+      description: password of SQL user
+      x-k8s-cli:
+        setter:
+          name: password
+          value: ${PASSWORD?}
+          setBy: PLACEHOLDER
+    io.k8s.cli.setters.database-name:
+      description: name of SQL database
+      x-k8s-cli:
+        setter:
+          name: database-name
+          value: mysql-ha-database
+          setBy: package-default
+    io.k8s.cli.setters.instance-name:
+      description: name of SQL instance
+      x-k8s-cli:
+        setter:
+          name: instance-name
+          value: mysql-ha-instance
```

Comment:
I don't think a placeholder is appropriate here because the instance name does not need to be globally unique. I will add a note about the names being kept to the README.

Reply:
Okay, makes sense. Then can you change the default name to `mysql-ha-solution`? This will have less chance to cause conflict.